### PR TITLE
Move transaction option from Settings to project properties

### LINKED ISSUE #952: [feature] allow generating default values on postgres just in time
Original commit: https://github.com/qgis/QGIS/commit/d1c79d817c2527367de48a06a235a24c41979068 by m-kuhn

The expression used to generate default values for a field on the
postgres database are currently being printed on the feature form.

    nextval('serial')

A new option in the properties allows evaluating the expression before
usage and will directly print the new value in the feature form.

    23

### PR DESCRIPTION
**Pull request by DelazJ**
as discussed in https://github.com/qgis/QGIS-Documentation/issues/758#issuecomment-215051691
Also fix #952

### CODE REVIEW

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 1]
```diff
@@ -112,6 +112,21 @@ can set project-specific options.
   and opening its form) will list the related entities. This provides a powerful
   way to express e.g. the inspection history on a length of pipeline or road segment.
   You can find out more about 1:n relations support in Section :ref:`vector_relations`.
+* In the :guilabel:`Data Sources` menu, you can:
```
Comment (yjacolin) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: could we use `tab` instead of `menu`?

**Comment by yjacolin** [comment]
LGTM

File: source/docs/user_manual/introduction/qgis_configuration.rst (hunk 1 above)
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: I kept menu to be coherent with the description of the Options dialog in this file: all tabs are called menus.
Will fix all of them them.

**Comment by yjacolin** [comment]
Thanks. I let you merge when you think it is ready.

**Comment by DelazJ** [comment]
Done, Thanks!

**Pull request merged**